k8s/templates/filings-ui.dc-srvc-rt-cmap-kcmap.json

```diff
@@ -0,0 +1,385 @@
+            },
+            "metadata": {
+                "name": "${NAME}-${TAG_NAME}-configuration",
+                "labels": {
+                    "app": "${NAME}-${TAG_NAME}",
+                    "app-group": "${APP_GROUP}-${TAG_NAME}"
+                }
+            }
+        },
+        {
+            "kind": "Service",
+            "apiVersion": "v1",
+            "metadata": {
+                "labels": {
+                    "app": "${NAME}-${TAG_NAME}"
+                },
+                "name": "${NAME}-${TAG_NAME}"
+            },
+            "spec": {
+                "ports": [
+                    {
+                        "name": "8080-tcp",
+                        "port": 8080,
+                        "protocol": "TCP",
+                        "targetPort": 8080
+                    }
+                ],
+                "selector": {
+                    "deploymentconfig": "${NAME}-${TAG_NAME}"
+                },
+                "sessionAffinity": "None",
+                "type": "ClusterIP"
+            }
+        },
+        {
+            "apiVersion": "route.openshift.io/v1",
+            "kind": "Route",
+            "metadata": {
+                "labels": {
+                    "app": "${NAME}-${TAG_NAME}",
+                    "app-group": "${APP_GROUP}-${TAG_NAME}"
+                },
+                "name": "filings-ui-${TAG_NAME}"
+            },
+            "spec": {
+                "host": "${DOMAIN}",
+                "path": "${PATH}",
+                "port": {
+                    "targetPort": "8080-tcp"
+                },
+                "tls": {
+                    "insecureEdgeTerminationPolicy": "Redirect",
+                    "termination": "edge"
+                },
+                "to": {
+                    "kind": "Service",
+                    "name": "${NAME}-${TAG_NAME}",
+                    "weight": 100
+                },
+                "wildcardPolicy": "None"
+            },
+            "status": {
+                "ingress": [
+                    {
+                        "conditions": [
+                            {
+                                "type": "Admitted"
+                            }
+                        ],
+                        "host": "${DOMAIN}",
+                        "routerName": "router",
+                        "wildcardPolicy": "None"
+                    }
+                ]
+            }
+        }
+    ],
+    "parameters": [
+        {
+            "name": "NAME",
+            "displayName": "Name",
+            "description": "The name assigned to all of the OpenShift resources associated to this app.",
+            "required": true,
+            "value": "business-create-ui"
```

Comment:
Do some of these parameter values need to be changed? This one says business-create-ui.

Reply:
Right - that's not the actual value just the placeholder (so it's not actually used), but I will update that

Reply:
updated